### ISSUE TITLE
1.3 beta 7 Uncaught TypeError: process.nextTick is not a function

### ISSUE THREAD

**Issue by rclai** [meteor/meteor#6167]
I'm getting this when using

```
react-formal
react-formal-inputs
```

__Edit__: This is because Meteor has not provided a client polyfill for `process.nextTick`.

**Comment by rclai** [comment]
@benjamn this is coming from the `modules` package.

**Comment by rclai** [comment]
My current workaround is to do this in client code space:

```js
process.nextTick = Meteor.defer;
```

**Comment by avital** [comment]
Will be resolved by #6056

**Comment by benjamn** [comment]
Fixed in [`beta.12`](https://github.com/meteor/meteor/issues/6266#issuecomment-190789852).
